vpr/src/base/vpr_types.h

```diff
@@ -837,6 +837,11 @@ enum e_place_algorithm {
     PATH_TIMING_DRIVEN_PLACE
 };
 
+enum e_pad_loc_type {
+	FREE,
```

Comment:
Give a short comment of FREE and RANDOM (I/Os can move and optimized in placement, and I/Os are randomly locked down (to emulate board-level constraints).